src/Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/GaussianPlusConstant.hpp

```diff
@@ -0,0 +1,119 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines GeneralizedHarmonic::ConstraintDamping::GaussianPlusConstant.
+
+#pragma once
+
+#include <array>
+#include <pup.h>
+
+#include "Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/GhDampingFunction.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace GeneralizedHarmonic::ConstraintDamping {
+/*!
+ * \brief GaussianPlusConstant \f$f = C + A
+ * \exp\left(-\frac{(x-x_0)^2}{w^2}\right)\f$
+ *
+ * \details Input file options are: `Constant` \f$C\f$, `Amplitude` \f$A\f$,
+ * `Width` \f$w\f$, and `Center`\f$x_0\f$. The function takes input coordinates
+ * of type `tnsr::I<T, VolumeDim, Fr>`, where `T` is e.g. `double` or
+ * `DataVector`, `Fr` is a frame (e.g. `Frame::Inertial`), and `VolumeDim` is
+ * the dimension of the spatial volume.
+ */
+template <size_t VolumeDim, typename Fr>
+class GaussianPlusConstant : public GhDampingFunction<VolumeDim, Fr> {
+ public:
+  struct Constant {
+    using type = double;
+    static constexpr Options::String help = {"The constant."};
+  };
+
+  struct Amplitude {
+    using type = double;
+    static constexpr Options::String help = {"The amplitude of the Gaussian."};
+  };
+
+  struct Width {
+    using type = double;
+    static constexpr Options::String help = {"The width of the Gaussian."};
+    static type lower_bound() noexcept { return 0.; }
+  };
+
+  struct Center {
+    using type = std::array<double, VolumeDim>;
+    static constexpr Options::String help = {"The center of the Gaussian."};
+  };
+  using options = tmpl::list<Constant, Amplitude, Width, Center>;
+
+  static constexpr Options::String help = {
+      "Computes a Gaussian plus a constant about an arbitrary coordinate "
+      "center with given width and amplitude"};
+
+  WRAPPED_PUPable_decl_base_template(
```

Comment:
A bit of a nitpick, but I noticed that `WRAPPED...` and the `CkMigrateMessage` constructor show up in the Doxygen (probably true in other places). It would be cleaner if we `/// \cond` them, I think.